### PR TITLE
Tooltip + animation

### PR DESCRIPTION
**Pull request by CozlovschiNichita**
# Tooltip + animation "slideFromLeft"

https://github.com/CozlovschiNichita/Tekwill_FE_gr11/assets/161749799/9e53da83-5a33-41a6-9760-170b40031c12

### CODE REVIEW

**Review by dragan-andrei — APPROVED** [review]
Super very good :3

File: tooltip.css [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+/* Стили для каждого триггера с тултипом */
+.tooltip {
+    position: relative; /* Позиционируем относительно своего родителя */
+    display: inline-block; /* Блочный элемент с возможностью выравнивания по горизонтали */
+    margin: 20px; /* Отступы вокруг элемента */
+    cursor: pointer; 
+    border: 2px solid rgb(167, 0, 0); 
+}
+
+.tooltip .tooltipleft, .tooltip .tooltiptop, .tooltip .tooltiptbottom, .tooltip .tooltipright {
```
Comment (IvanCebanPVT) [inline tooltip.css]: Good for optimizing the styles!

You could go even further with that and just add all of them a common selector, like ```.tooltip span {}``` and set these styles to each span that is in .tooltip

Or even better give a common class to these spans, so you will have for example ```<span class="tooltip_content tooltipleft">Tooltip Left</span>``` and set all common styles to class "tooltip_content"

File: tooltip.css [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    text-align: center; 
+    border-radius: 5px; 
+    padding: 5px 0; /* Отступы внутри тултипа */
+    position: absolute; 
+    z-index: 1; /* Поверх всех остальных элементов */
+    opacity: 0; /* Изначально тултип невидимый */
+    transition: opacity 0.3s; /* Плавное изменение прозрачности */
+}
+
+.tooltip .tooltipleft::after, .tooltip .tooltiptop::after, .tooltip .tooltiptbottom::after, .tooltip .tooltipright::after {
```
Comment (IvanCebanPVT) [inline tooltip.css]: Same idea here. If you give all tooltips common class you could set these styles to ```. tooltip_content::after {}``` and that's it.

File: tooltip.css [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+.tooltip .tooltipright::after {
+    margin-top: -5px; /* Отступ стрелки сверху */
+    border-color: transparent #555 transparent transparent; 
+    top: 50%; /* Положение стрелки относительно верхней границы тултипа */
+    right: 100%; /* Положение стрелки относительно правой границы тултипа */
+}
+
+/* Отображение тултипов при наведении */
+.tooltip:hover .tooltipleft, .tooltip:hover .tooltiptop, .tooltip:hover .tooltiptbottom, .tooltip:hover .tooltipright {
```
Comment (IvanCebanPVT) [inline tooltip.css]: Same idea here as from above

**Review by IvanCebanPVT — APPROVED** [review]
Very well done!
There is an optimization suggestion, which I recommend to apply. 
And you're good to go

**Review by IvanCeban — APPROVED** [review]

**Comment by CozlovschiNichita** [comment]
we'll fix it

**Pull request merged**